src/contracts/interfaces/IEigenPod.sol

```diff
@@ -50,6 +50,13 @@ interface IEigenPod {
         VALIDATOR_STATUS status;
     }
 
+    struct WithdrawalInfo {
+        //amount to send to a podOwner from a proven withdrawal
+        uint256 amountToSend;
+        //difference in shares to be recorded in the eigenPodManager
+        int256 sharesDelta;
+    }
```

Comment:
is there something else we can name this, perhaps more specific or descriptive?
even just adding a top-level comment to the struct definition (perhaps just capturing its usage) might help.
also going to note a nit about spacing; should have a space between the slashes and start of comment

Reply:
suggestion:
```
    /** 
     * @notice struct used to store amounts related to proven withdrawals in memory. Used to help
     * manage stack depth and optimize the number of external calls, when batching withdrawal operations.
     */
    struct ProvenWithdrawal {
        // amount to send to a podOwner from a proven withdrawal
        uint256 amountToSend;
        // difference in shares to be recorded in the eigenPodManager, as a result of the withdrawal
        int256 sharesDelta;
    }
```

Reply:
fixed